src/lib/mongoBackend/mongoNotifyContextAvailability.cpp

```diff
@@ -33,10 +33,16 @@
 
 /* ****************************************************************************
 *
-* mongoNofityContextAvailability -
+* mongoNotifyContextAvailability -
 */
-HttpStatusCode mongoNotifyContextAvailability(NotifyContextAvailabilityRequest* requestP, NotifyContextAvailabilityResponse* responseP, const std::string& tenant) {
-
+HttpStatusCode mongoNotifyContextAvailability
```

Comment:
Looking to these modifications to mongoNotifyContextAvailability related files, I wonder if the servicePath in notifications is also supported after this PR. Although it is not part of the functionality described https://github.com/telefonicaid/fiware-orion/issues/719 if we can have it also, of course it would be great :)